src/Components/Sponsor.js

```diff
@@ -0,0 +1,174 @@
+import "../CSS/Sponsor.css";
+import "../CSS/Officers.css";
+import React, { useState } from "react";
+
+function Sponsor() {
+  const [ContentSelected, setContentSelected] = useState("sponsor");
+  const [contactData, setContactData] = useState({
+    name: "",
+    email: "",
+    company: "",
+    message: "",
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const { name, email, company, message } = contactData;
+    window.location.href = `mailto:sweccmailinglist@u.washington.edu?subject=
+    New Contact from ${name}&body=Name: ${name} from ${company} %0D%0AEmail: ${email}%0D%0AMessage: ${message}`;
+    setTimeout(() => {
+      if (document.hasFocus()) {
+        alert(
+          "It seems your email client didn't open automatically. Please send an email to y031125k@gmail.com with your message.",
```

Comment:
What email is this? Shouldn't we say to email the swecc email?

Reply:
+1